udfs/flex/text/indexOf.md

```
@@ -0,0 +1,67 @@
+
+The zero-based index of the first occurrence of the substring, or `-1` if not found. Returns `null` if the input string is `null`.
+
+## Examples
+
+### Example 1: Basic Search
+```cypher
+RETURN flex.text.indexOf('hello world', 'world') AS position
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)